tests/fakeprotocol_smart.py

```diff
@@ -444,10 +450,10 @@ async def _send_request(self, request_dict: dict):
             return await self._handle_control_child(request_dict["params"])
 
         params = request_dict.get("params", {})
-        if method == "component_nego" or method[:4] == "get_":
+        if method in {"component_nego", "qs_component_nego"} or method[:4] == "get_":
             if method in info:
                 result = copy.deepcopy(info[method])
-                if "start_index" in result and "sum" in result:
+                if result and "start_index" in result and "sum" in result:
```

Comment:
result is None in some of our fixtures, i.e. for `get_inherit_info`

Reply:
Would it make more sense to bail out as early as possible, i.e., inverse the checks? Could we avoid the need for having a `verbatim` then?

Reply:
`verbatim` is preventing *all* the missing fixture data logic because the purpose of the `dump_devinfo` test is to ensure that the fixtures are recreated the same.